Makefile.am

```diff
@@ -4,13 +4,24 @@
 
 ACLOCAL_AMFLAGS = -I m4
 
+if BUILD_OWN_PTHREAD_WORKQUEUES
```

Comment:
I would rather write this:

```
if BUILD_OWN_PTHREAD_WORKQUEUES
PTHREAD_WORKQUES_SUBDIR = libpwq
else
PTHREAD_WORKQUES_SUBDIR =
endif
```

and add that variable in SUBDIRS, else maintaining the list below will be a pain. or is it not working?